packages/attribute-registry-process/test/db.test.ts

```diff
@@ -73,103 +78,189 @@ describe("database test", () => {
+        // TO DO
+        expect(1).toBe(1);
+      });
+      it("should not write on event-store if the attribute already exists", () => {
+        // TO DO
+        expect(1).toBe(1);
+      });
+      it("should not write on event-store if the organization is not a certifier", () => {
+        // TO DO
+        expect(1).toBe(1);
+      });
+      it("should not write on event-store if the certifier is not found", () => {
+        // TO DO
+        expect(1).toBe(1);
+      });
     });
-  });
 
-  describe("get an attribute by name", () => {
-    it("should get the attribute if it exists", async () => {
-      const id = uuidv4();
-      await addOneAttribute(id);
-      const attribute = await readModelService.getAttributeByName("name");
-      expect(attribute?.data.name).toBe("name");
-      expect(attribute?.data.description).toBe("description");
-    });
-    it("should not get the attribute if it doesn't exist", async () => {
-      const attribute = await readModelService.getAttributeByName("name");
-      expect(attribute).toBeUndefined();
+    describe("readModelService", () => {
+      describe("getAttributesByIds", () => {
+        it("should get the attributes if they exist", async () => {
+          // TO DO
+          expect(1).toBe(1);
+        });
+        it("should not get the attributes if they don't exist", async () => {
+          // TO DO
+          expect(1).toBe(1);
+        });
+        it("should not get any attributes if the requested ids list is empty", async () => {
+          // TO DO
+          expect(1).toBe(1);
+        });
+      });
+      describe("getAttributesByKindsNameOrigin", () => {
+        it("should get the attributes if they exist (parameters: kinds, name, origin)", () => {
+          // TO DO
+          expect(1).toBe(1);
+        });
+        it("should get the attributes if they exist (parameters: kinds only)", () => {
+          // TO DO
+          expect(1).toBe(1);
+        });
+        it("should get the attributes if they exist (parameters: name only)", () => {
+          // TO DO
+          expect(1).toBe(1);
+        });
+        it("should get the attributes if they exist (parameters: origin only)", () => {
+          // TO DO
+          expect(1).toBe(1);
+        });
+        it("should get all the attributes if no parameter is passed", () => {
+          // TO DO
+          expect(1).toBe(1);
+        });
+        it("should not get the attributes if they don't exist", () => {
+          // TO DO
+          expect(1).toBe(1);
+        });
+      });
+      describe("get an attribute by id", () => {
+        it("should get the attribute if it exists", async () => {
+          await addOneAttribute(mockAttribute);
+          const attribute = await readModelService.getAttributeById(
+            mockAttribute.id
+          );
+          expect(attribute?.data.name).toBe(mockAttribute.name);
+          expect(attribute?.data.description).toBe(mockAttribute.description);
```

Comment:
```suggestion
          expect(attribute?.data).toEqual(mockAttribute);
```

Reply:
Actually I wanted to do the change in the next pull request which contains more tests implementation. Anyway, updated here too.